### ISSUE TITLE
Issue regarding import

### ISSUE THREAD

**Issue by jmkim93** [taichi-dev/taichi#7904]
**Describe the bug**
After installing Taichi in conda environment on my Apple Silicon Mac using 
```
pip install taichi
```
I encountered an import issue as follows.

**To Reproduce**
```py
import taichi
```

**Log/Screenshots**

```
runcell(0, '/Users/XXXX/Documents/ONN/Practices/taichi_ex.py')
Share object taichi_python import failed, check this page for possible solutions:
https://docs.taichi-lang.org/docs/install
Traceback (most recent call last):

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/spyder_kernels/py3compat.py:356 in compat_exec
    exec(code, globals, locals)

  File ~/Documents/ONN/Practices/taichi_ex.py:1
    import taichi as ti

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/__init__.py:3
    from taichi._funcs import *

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_funcs.py:3
    from taichi.lang import impl, ops

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/lang/__init__.py:1
    from taichi.lang import impl, simt

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/lang/impl.py:5
    from taichi._lib import core as _ti_core

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/__init__.py:1
    from taichi._lib.utils import ti_python_core as core

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/utils.py:110
    ti_python_core = import_ti_python_core()

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/utils.py:55 in import_ti_python_core
    raise e from None

  File /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/utils.py:45 in import_ti_python_core
    from taichi._lib.core import \

ImportError: dlopen(/opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/core/taichi_python.cpython-310-darwin.so, 0x000A): Library not loaded: /opt/homebrew/opt/zstd/lib/libzstd.1.dylib
  Referenced from: <3DE40B11-788F-3921-8F90-38A5B48C4702> /opt/homebrew/Caskroom/miniforge/base/envs/onn/lib/python3.10/site-packages/taichi/_lib/core/taichi_python.cpython-310-darwin.so
  Reason: tried: '/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/local/lib/libzstd.1.dylib' (no such file), '/usr/lib/libzstd.1.dylib' (no such file, not in dyld cache)
```

**Additional comments**
Can I get a comment on this issue?
Thanks.

**Comment by bobcao3** [comment]
Can you try to remove or reinstall libzstd in homebrew?

**Comment by feisuzhu** [comment]
Sorry about the late reply, it's a known issue on M1 wheels which will be fixed in v1.6.0.

You can workaround this by `brew install zstd`

**Comment by feisuzhu** [comment]
v1.6.0 is released
